applications/nrf_desktop/doc/hids.rst

```diff
@@ -72,7 +72,7 @@ HID keyboard LED output report
 
 The module can receive an HID output report setting state of the keyboard LEDs, e.g. state of the Caps Lock.
 The report is received from the Bluetooth connected host.
-The module forwards the report using ``hid_report_event``, that is handled either by |hid_state| (for peripheral) or :ref:`nrf_desktop_hid_forward` (for central).
+The module forwards the report using ``hid_report_event``, that is handled by |hid_state|.
```

Comment:
```suggestion
The module forwards the report using ``hid_report_event`` that is handled by |hid_state|.
```
FYI: I removed all the commas used in similiar context as these are restrictive clauses.